File: output/ChatGPT/avatar/Java/Python/atcoder_ARC062_B.py
import sys

class TaskD:
    def solve(self, S):
        score = 0
        for i in range(len(S)):
            my = 'g' if i % 2 == 0 else 'p'
            his = S[i]
            if my != his:
                score += 1 if my == 'p' else -1
        return score

if __name__ == "__main__":
    S = input()
    solver = TaskD()
    score = solver.solve(S)
    print(score)

File: output/ChatGPT/avatar/Java/C++/atcoder_ARC062_B.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    string S;
    cin >> S;
    int score = 0;
    for (int i = 0; i < S.length(); i++) {
        char my = i % 2 == 0 ? 'g' : 'p';
        char his = S[i];
        if (my != his) {
            score += my == 'p' ? 1 : -1;
        }
    }
    cout << score << endl;
}